hbase-annotations/pom.xml

```diff
@@ -54,3 +54,4 @@
     </plugins>
   </build>
 </project>
+
```

Comment:
should revert at all places

Reply:
Why bunch of this new line change?

Reply:
i wanted to trigger all tests for this PR, so modified all files to add a dummy whitespace to each pom. Have reverted since build is GREEN now